### DISCUSSION TITLE
Welcome to keygen Discussions!

### DISCUSSION THREAD

**Discussion by charely6** [charely6/keygen#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AshSabine** [comment]
I've been working on adding a whole list of BEST wardings to the key generator - all excluding the W series, B1 & B2 (although I could probably add those in short order). Haven't found `make` (as is listed as part of the generation process) anywhere, is it part of the `Makefile` or do I have to use a utility like paths2openscad?

**Reply by charely6** [reply]
Ah I might have to add some clarification for that. 
Make is program on your computer that uses the Makefile to go through and do stuff usually for compiling software.
if you open a cmdline or terminal (linux or maybe Mac?) and go to where you download/clone the repo you should be able to just run make in the same folder that the makefile is in and it will go through and do everything.
